pom.xml

```diff
@@ -602,6 +603,10 @@
                                 <ignoreDestinationPackage>
                                     <package>com.jcraft.jzlib</package>
                                 </ignoreDestinationPackage>
+                                <!-- ignore references to Android util in okhttp3 -->
+                                <ignoreDestinationPackage>
+                                    <package>android.util</package>
```

Comment:
If you think this is a "safe" thing to ignore, which I assume it is, I'm totally fine with this.